DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -73,21 +78,6 @@ private string GetScoreSummary(int currentScore, int possibleScore)
             return CanShowProgress ? $"(score: {currentScore} out of {possibleScore})" : "";
         }
 
-        private string GetTimeSummary(int timeSpent, int averageTutorialDuration)
-        {
-            return $"{ParseMinutes(timeSpent)}" +
-                   $" (average time {ParseMinutes(averageTutorialDuration)})";
-        }
-
-        private static string ParseMinutes(int minutes)

```

Comment:
Instead of not doing this on either page, I think we should do it on both. It definitely looks better when the time is longer than an hour 🙂

Reply:
Yeah I agree. I remembered some code I wrote for the initial menu did this well, so I've refactored that and used it in this case as well 🙂